.bcr/config.yml

```diff
@@ -0,0 +1,3 @@
+fixedReleaser:
```

Comment:
Workaround for this [issue](https://github.com/bazel-contrib/publish-to-bcr#a-note-on-release-automation)